stock_picking_auto_print/models/ir_actions_report.py

```diff
@@ -0,0 +1,13 @@
+# Copyright (C) 2019 IBM Corp.
+# Copyright (C) 2019 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class IrActionsReport(models.Model):
+    _inherit = "ir.actions.report"
+
+    is_default_report = fields.Boolean('Is Default Report?')
```

Comment:
```suggestion
    is_default_report = fields.Boolean('Is Default Report?', default=False)
```

Reply:
@bodedra I am guessing there should only be one default per country or company, no?

Reply:
IMO, Boolean is by default is false. Ref https://github.com/odoo/odoo/blob/11.0/odoo/models.py#L3478

Yes. We can add constraint for one default per country AND company. Regarding per country OR company, may we leave it to decide User? Currently, this module will fetch only one records set. Or any other method to restrict except create()/write() method?